modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Questions.es.js

```diff
@@ -345,20 +345,21 @@ export default withRouter(
 					pageSize,
 					search,
 					section,
-					siteKey
+					siteKey,
+					'dateCreated:desc'
 				);
 			}
 
-			fn.then(({data, loading}) => {
+			fn.then(({data}) => {
 				setQuestions(data || []);
-				setLoading(loading);
-			}).catch((error) => {
-				if (process.env.NODE_ENV === 'development') {
-					console.error(error);
-				}
-				setLoading(false);
-				setError({message: 'Loading Questions', title: 'Error'});
-			});
+			})
+				.catch((error) => {
```

Comment:
I don't like the formatting at all but seems that is like the Source Formatting does with the finally